src/fastoad_cs25/models/weight/mass_breakdown/cs25.py

```diff
@@ -34,6 +34,9 @@ class Loads(ExplicitComponent):
 
     """
 
+    def initialize(self):
+        self.options.declare("fuel_load_alleviation", types=bool, default=True)
+
```

Comment:
Despite the rather explicit name, I would appreciate to have the `desc=` field.

Reply:
Fixed in [d0f3099](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/38/commits/d0f3099109c8fb3f4c9d1e683b98a6755a0e8e9d)